### ISSUE TITLE
error:"dataRoot parameter is not set in configuration.xml!"

### ISSUE THREAD

**Issue by zhangchunlin** [OpenGrok/OpenGrok#863]
opengrok-0.12.1
apache-tomcat-7.0.56
java version "1.7.0_65" (OpenJDK Runtime Environment (IcedTea 2.5.2) (7u65-2.5.2-3~14.04))

When click "search",I got error:
```
There was an error!

CONFIGURATION parameter has not been configured in web.xml! Please configure your webapp.

dataRoot parameter is not set in configuration.xml!

java.security.InvalidParameterException: dataRoot parameter is not set in configuration.xml!
	at org.opensolaris.opengrok.web.PageConfig.getDataRoot(PageConfig.java:1142)
	at org.opensolaris.opengrok.web.PageConfig.prepareSearch(PageConfig.java:1167)
	at org.apache.jsp.search_jsp._jspService(search_jsp.java:157)
```

But in etc/configuration there is dataRoot:
```
<code>
  <void property="dataRoot">
   <string>/home/zhangclb/1del/opengrok_depl/opengrok-instance/data</string>
  </void>
</code>
```

I don't know how to fix this issue.
I try opengrok-0.11.1 but still the same.

**Comment by Ricky54326** [comment]
@tarzanek  I'm getting this error when trying to install on windows using the guide linked in the readme. Any ideas?

**Comment by tarzanek** [comment]
on windows the default path that default source.war reads would be C:\var\opengrok\etc\confguration.xml
so your (default) DATA_ROOT should be C:\var\opengrok
if case you changed DATA_ROOT, you need to modify the source.war and adjust web.xml ...

**Comment by tarzanek** [comment]
should be fixed in #1026, closing
